extensions/amp-analytics/0.1/vendors/subscriptions-propensity.js

```diff
@@ -19,19 +19,26 @@ import {jsonLiteral} from '../../../../src/json';
 const SUBSCRIPTIONS_PROPENSITY_CONFIG = jsonLiteral({
   'vars': {
     'clientId': 'CLIENT_ID(__gads)',
-    'data':
-      '{"skus": "${skus}","source": "${source}","active": "${active}","product": "${product}"}',
+    'startJson': '%7B',
+    'endJson': '%7D',
+    'q': '%22',
+    'c': '%3A',
+    ',': '%2C%20',
```

Comment:
`', '` instead? Should we separate it to `,` and `s`?

Reply:
I was mostly trying to create reusable tokens that masked the ugly pre-encoded string values.  That comma is being used to separate key/value pairs.

I could separate it into a , and a s but it expands the number of tokens required in a few places.  I'd rather not have a space after the comma (I think this is acceptable JSON formatting wise?) than have a second token.  It does appear in the .amp-analytics config on the publishers .amp page.